packages/camera/src/components/Controls/index.js

```diff
@@ -56,9 +56,12 @@ export default function Controls({
   addDamageParts,
   onResetAddDamageStatus,
   onCloseEarly,
+  isPortraitModeVinLayoutView,
   ...passThroughProps
 }) {
   const { height: windowHeight } = useWindowDimensions();
+  const portraitModeRotationControls = [Controls.SettingsButtonProps.id,
```

Comment:
You can add `rotate: true` in `Controls.CaptureButtonProps` and other control objects and use it directly instead of creating this new array and checking the control in this collection to apply rotate property